acvm/src/pwg/directives.rs

```diff
@@ -92,6 +92,34 @@ pub fn solve_directives(
 
             Ok(())
         }
+        Directive::ToRadixBe { a, b, radix } => {
+            let val_a = get_value(a, initial_witness)?;
+
+            let a_big = BigUint::from_bytes_be(&val_a.to_be_bytes());
+            let a_dec = a_big.to_radix_be(*radix);
```

Comment:
How would it look if we had a utility function for both `ToRadixBe` and `ToRadixLe` after this line? It looks like the exact same code and the only info that would need to be passed is `a_dec`, `b`, and `initial_witness`.

Reply:
makes sense